docs/faq.rst

```diff
@@ -115,6 +122,34 @@ See above.
 
 ------
 
+**Q. Why do I see error code 503 with the message “Error: configuration operation in progress on device, Please try again in 2 min” on the 'Pending tasks' table of the app?**
```

Comment:
(not sure if this is the actual error message, but the capital letters seem off and needs an article at the beginning.. if this is the actual error message, ignore for now, and maybe fix the error message and this text?)
"Error: A configuration operation is in progress on the device, try again in 2 minutes"

Reply:
Hi @JosephPJordan : This error comes from the BIG-IP directly and is just displayed by the app as is: Error: Configuration operation in progress on device <BIG-IP IP>, Please try again in 2 minutes. I've updated with the exact message now.